src/features/multi-step-form/ui/careerForm/index.tsx

```
@@ -0,0 +1,28 @@
+// 스텝에 따른 Fomr 컴포넌트를 렌더링하는 컴포넌트
+const StepFormView = ({ currentStep }: { currentStep: number }) => {
+  const stepComponents: Record<number, JSX.Element> = {
+    1: <FirstStep />,
+    2: <TwoStep />,
+    3: <ThirdStep />,
+  };
+
+  return stepComponents[currentStep] || <></>;
+}
+
+function CareerFormView({ currentStep }: CareerFormViewProps) {
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)